src/main/java/com/github/manolo8/darkbot/utils/DiscordWebhook.java

```diff
@@ -43,6 +45,10 @@ public void addEmbed(EmbedObject embed) {
         this.embeds.add(embed);
     }
 
+    public void addEmbed(Function<EmbedObject, EmbedObject> function) {
+        addEmbed(function.apply(new EmbedObject()));
+    }
+
```

Comment:
This is just stupid, it really does not help make code any shorter from the calling client

if you want to make this better-to-use, make builder-style setters:
```java
public DiscordWebhook setX(X x) {
    this.x = x;
    return this;
}
```
as this allows chaning calls from the client using the api:
```java
new DiscordWebhook()
    .setX("asdf")
    .setY("asdf2)
    .execute("url");
```

Reply:
so should I remove this way?